spring-cloud-gcp-samples/pom.xml

```diff
@@ -49,7 +49,8 @@
 		<module>spring-cloud-gcp-data-multi-sample</module>
 		<module>spring-cloud-gcp-data-firestore-sample</module>
 		<module>spring-cloud-gcp-bigquery-sample</module>
-	</modules>
+		<module>spring-cloud-gcp-pubsub-stream-functional-binder-sample</module>
```

Comment:
Looking at  `spring-cloud-gcp-pubsub-binder-sample` and `spring-cloud-gcp-pubsub-polling-binder-sample`,
Maybe it should be `spring-cloud-gcp-pubsub-functional-binder-sample` for consistency sake.

Reply:
I want to rename `spring-cloud-gcp-pubsub-binder-sample` and `spring-cloud-gcp-pubsub-polling-binder-sample`, actually. The original name of "pubsub binder" is confusing -- what is this binder we speak about?

The new names would be `spring-cloud-gcp-pubsub-stream-binder-sample` and `spring-cloud-gcp-pubsub-polling-stream-binder-sample`.

Or even ditch "binder" completely -- it's an implementation detail, completely irrelevant to end user.  So: `spring-cloud-gcp-pubsub-stream-sample` and `spring-cloud-gcp-pubsub-polling-stream-sample` and `spring-cloud-gcp-pubsub-stream-functional-sample`.

Reply:
I think as long as the actual module name that this is a sample for is called `spring-cloud-gcp-pubsub-stream-binder`, we should keep it as a base name for the samples, even if that means renaming some of the older ones.

BTW, as you do the renaming and adding new samples, please don't forget to add/edit the links in refdocs.